kobo/apps/reports/report_data.py

```diff
@@ -130,10 +130,15 @@ def data_by_identifiers(asset, field_names=None, submission_stream=None,
     if field_names is None:
         field_names = fields_by_name.keys()
     if split_by and (split_by not in fields_by_name):
-        raise serializers.ValidationError(_("`split_by` field '{}' not found.").format(split_by))
+        raise serializers.ValidationError({
+            'data_by_identifiers': _("`split_by` field '{}' not found.").format(split_by)
```

Comment:
i don't remember how this gets here, or even if this message would ever appear to a user. if it can't happen directly from user input, then maybe it should be a different exception that returns a 500, so that we're alerted to a programmer's mistake

Reply:
Actually it comes from the user, `split_by` is the value passed in the `GET` request as query string. 
e.g : https://kf.kobotoolbox.org/reports/{asset_id}/?names=&split_by={question_name}

`data_by_identifiers` is not the correct key though, it should be `split_by`